index.html

```diff
@@ -4124,6 +4135,60 @@ <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
         </section>
       </section>
 
+      <section id="dWLm-chunk">
+        <h2><span class="chunk">dWLm</span> Diffuse white luminance metadata</h2>
+        <p>The two-byte chunk type field contains the hexadecimal values</p>
+
+        <pre>
+<!-- xx xx xx xx
+</pre>
```

Comment:
I think you don't want to include these 3 lines (4142-4144).

Reply:
I think this was intended as a placeholder for the chunk type field